tests/testthat/test_pipelines.R

```diff
@@ -107,3 +107,69 @@ test_that("survtoregr 3", {
   expect_prediction_surv(p)
   expect_true("distr" %in% p$predict_types)
 })
+
+skip_if_not_installed("mlr3learners")
+
+test_that("survtoclassif", {
+  requireNamespace("mlr3learners")
+
+  pipe = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"))
+  expect_class(pipe, "Graph")
+  grlrn = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"),
+                             graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  grlrn$train(task)
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
+
+  cox = lrn("surv.coxph")
+  suppressWarnings(cox$train(task))
+  p2 = cox$predict(task)
+
+  expect_equal(p$truth, p2$truth)
+  expect_equal(p$score(), p2$score(), tolerance = 0.1)
+
+  # Test with cut
+  grlrn = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"),
+                            cut = c(10, 30, 50), graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  suppressWarnings(grlrn$train(task))
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
+
+  # Test with max event time
+  max_time = task$data()[status == 1, min(time)]
+  grlrn = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"),
+                             max_time = max_time, graph_learner = TRUE)
+  expect_error(grlrn$train(task))
+
+  grlrn = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"),
+                             max_time = max_time + 1, graph_learner = TRUE)
+  suppressWarnings(grlrn$train(task))
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
+
+  # Test with rhs
+  grlrn = ppl("survtoclassif", learner = lrn("classif.log_reg"), rhs = "1",
+              graph_learner = TRUE)
+  grlrn$train(task)
+  pred = grlrn$predict(task)
+
+  grlrn = ppl("survtoclassif", learner = lrn("classif.featureless"), graph_learner = TRUE)
+  grlrn$train(task)
+  pred2 = grlrn$predict(task)
+
+  expect_equal(pred$data$distr, pred2$data$distr)
+
+  grlrn = ppl("survtoclassif", learner = lrn("classif.log_reg"), rhs = "rx + litter",
+             graph_learner = TRUE)
+  grlrn$train(task)
+  pred = grlrn$predict(task)
+
+  grlrn2 = ppl("survtoclassif", learner = lrn("classif.log_reg"), rhs = ".",
+               graph_learner = TRUE)
+  grlrn2$train(task)
+  pred2 = suppressWarnings(grlrn2$predict(task))
+
+  expect_lt(pred$score(), pred2$score())
```

Comment:
Is there a reason that we expect the cindex to be less here using the different formulas?

Reply:
Model with covariates should perform better than without

Reply:
to discuss, I see one is `rhs = "."` the other `rhs = "rx + litter"` maybe we wanted to test against rhs = "1"`?